src/CADL.Extension/Emitter.Csharp/src/lib/model.ts

```diff
@@ -279,19 +294,7 @@ export function getInputType(
             }
         }
 
-        if (m.name === intrinsicName) {
-            // if the model is one of the Cadl Intrinsic type.
-            // it's a base Cadl "primitive" that corresponds directly to an c# data type.
-            // In such cases, we don't want to emit a ref and instead just
-            // emit the base type directly.
-            return {
-                Name: mapCadlIntrinsicModelToCsharpModel(program, m) ?? m.name,
```

Comment:
What is the reason behind to remove 'mapCadlIntrinsicModelToCsharpModel'

Reply:
I think this code will not be hit. So I deleted it.